packages/core/src/components/components.md

```diff
@@ -11,27 +13,49 @@
 @page collapse
 @page collapsible-list
 @page context-menu
-@page dialog
 @page editable-text
-@page forms
-@page html
 @page hotkeys
 @page icon
 @page menu
 @page navbar
 @page non-ideal-state
-@page overlay
-@page popover
-@page portal
 @page progress-bar
 @page skeleton
 @page spinner
-@page sliders
 @page table-html
 @page tabs
 @page tag
-@page tag-input
 @page text
+@page tree
+
+@## Forms
+
+@page form-group
+@page control-group
+
+<!-- ordering of these guys needs some work... what makes sense?? -->
+@page input
+@page input-group
+@page file-input
+@page numeric-input
+@page tag-input
```

Comment:
tried to group all the various inputs, but the `Blah Input` naming looks messy in nav.